roles/openshift_node/defaults/main.yml

```diff
@@ -45,10 +45,8 @@ openshift_node_syscon_add_mounts_l: []
 
 default_r_openshift_node_image_prep_packages:
```

Comment:
This can probably be removed in it's entirety now that everything is bootstrapped.

But, outside the scope of this PR.